packages/flutter_riverpod/lib/src/core/provider_scope.dart

```diff
@@ -347,3 +347,42 @@ To fix this problem, you have one of two solutions:
     return super.build();
   }
 }
+
+/// Widget testing helpers for flutter_riverpod.
+@visibleForTesting
+extension WidgetTesterHelpers on flutter_test.WidgetTester {
+  /// Attempt to look up the top-most [ProviderContainer] in the current widget.
+  ///
+  /// If `of` is provided is equivalent to
+  /// ```dart
+  /// final element = tester.element(of);
+  /// return ProviderScope.containerOf(element);
+  /// ```
```

Comment:
I don't think that's useful information.

For starter, we likely could deprecate `containerOf`